drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/DetectConflictingRowsCheck.java

```diff
@@ -47,15 +58,16 @@ public void check() {
     }
 
     @Override
-    public boolean isActive( final CheckWhiteList whiteList ) {
-        return whiteList.getAllowedCheckTypes()
-                .contains( CHECK_TYPE );
+    public boolean isActive( final CheckConfiguration checkConfiguration ) {
```

Comment:
The implementation of `isActive` method is the same in 3/5 occurrences. Maybe it would be worth to extract it into some parent class? What do you think @Rikkola ?